dist/index.html

```diff
@@ -7,6 +7,166 @@
   <body>
     <section class="all-recipes">
       <h1> Our recipes </h1>
+      <section class = "tag-area">
+        <div class="tag-rows">
+          <div class="tag-row row-one">
+            <div class = "tag-card">
```

Comment:
Screen readers may need to be able to identify these divs with the class of tag card. Can we change these to sections?

Reply:
Done